helm-charts/FATE/templates/nginx.yaml

```diff
@@ -133,11 +133,21 @@ spec:
         - name: nginx
           image: {{ .Values.image.registry }}/nginx:{{ .Values.image.tag }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/bash
+            - -c
+            - |
+              sed -i "s#conf/route_table.yaml#conf/modifiable/route_table.yaml#g" nginx/lua/route_table.lua;
```

Comment:
please remove this command, directly update "nginx/lua/route_table.lua" instead.